packages/lesswrong/components/search/SearchBarResults.tsx

```diff
@@ -79,60 +76,43 @@ const SearchBarResults = ({closeSearch, currentQuery, classes}: {
   currentQuery: string,
   classes: ClassesType
 }) => {
-  const { PostsSearchHit, UsersSearchHit, TagsSearchHit, CommentsSearchHit, Typography } = Components
+  const { PostsSearchHit, SequencesSearchHit, UsersSearchHit, TagsSearchHit, CommentsSearchHit, Typography } = Components
 
   return <div className={classes.root}>
     <div className={classes.searchResults}>
         <CurrentRefinements />
         <Components.ErrorBoundary>
-          <div className={classes.searchList}>
+          <div className={classes.usersList}>
+
             <Index indexName={getAlgoliaIndexName("Users")}>
-              <div className={classes.header}>
-                <Typography variant="body1">Users</Typography>
-                <Link to={`/search?terms=${currentQuery}`} className={classes.seeAll}>
-                  See all results
-                </Link>
```

Comment:
Looks like this link is gone now, but it's still useful to have since this dropdown limits the number of results. Maybe include it at the bottom of the list?